book/website/community-handbook/coworking/coworking-dropins.md

```diff
@@ -0,0 +1,90 @@
+# Drop-in Sessions
```

Comment:
Do we need chapter tags still? Check w/ infra group!

Reply:
They've never been "needed" (as in, the book still builds without them) but it's still good practice (they're needed if anyone ever wants to crosslink to that page) -
https://book.the-turing-way.org/community-handbook/style/style-crossref